analyses/interaction-plots/02-result-ns-for-manuscript.Rmd

```diff
@@ -58,9 +59,27 @@ n_hgg_sig <- hgg_results_sig %>%
   nrow()
 n_hgg_sig
 
-# Ratio of hgg:total significant events
-n_hgg_sig/n_sig
+# Find HGG gene pairs within All set
+# Note that this comparison is not statistically valid, 
+# as significance cutoffs are different, and comparing significant sets 
+# is problematic anyway
+
+# get pairs, alphabetically
+all_sig_genepairs <- all_results_sig %>%
+  mutate(pairs = case_when(gene1 < gene2 ~ paste(gene1, gene2, sep = "-"),
+                           gene1 > gene2 ~ paste(gene2, gene1, sep = "-"))) %>%
+  pull(pairs)
+
+hgg_sig_genepairs <- hgg_results_sig %>%
+  mutate(pairs = case_when(gene1 < gene2 ~ paste(gene1, gene2, sep = "-"),
+                           gene1 > gene2 ~ paste(gene2, gene1, sep = "-"))) %>%
+  pull(pairs)
+
+sum(hgg_sig_genepairs %in% all_sig_genepairs)
+```
 
+HGG mutual exclusivity counts
+```{r}
```

Comment:
I know you didn't write this chunk, but I think it could use a quick edit by adding this to the top - 
```
hgg_results %>%
    filter(gene1 == "PPM1D" & gene2 ==  "TP53")
```

This comes up with 0 rows, and provides some confirmation for anyone looking at the notebook that these genes are strictly in the given respective columns.